sdk/identity/identity/src/credentials/clientCertificateCredential.ts

```diff
@@ -28,23 +54,32 @@ export class ClientCertificateCredential implements TokenCredential {
    *
    * @param tenantId - The Azure Active Directory tenant (directory) ID.
    * @param clientId - The client (application) ID of an App Registration in the tenant.
-   * @param certificatePath - The path to a PEM-encoded public/private key certificate on the filesystem.
+   * @param configuration - Other parameters required, including the PEM-encoded certificate as a string, or as a path on the filesystem.
+   *                        If the type is ignored, we will throw if both the value of the PEM certificate and the path to a PEM certificate are provided at the same time.
    * @param options - Options for configuring the client which makes the authentication request.
    */
   constructor(
     tenantId: string,
     clientId: string,
-    certificatePath: string,
+    configuration: ClientCertificateCredentialPEMConfiguration,
```

Comment:
Can't this change be made without a breaking change? A separate overload perhaps?

Reply:
Yep, definitely. I will change it to avoid the breaking change.

Reply:
Updated!